src/BlazorUI/Bit.BlazorUI/Scripts/Utils.ts

```
@@ -23,81 +23,62 @@
         public static isTouchDevice() {
-            const matchMedia = window.matchMedia("(pointer: coarse)").matches;
-            const maxTouchPoints = ('ontouchstart' in window) || (navigator.maxTouchPoints > 0);
-
-            return matchMedia || maxTouchPoints;
+            try {
+                const matchMedia = window.matchMedia("(pointer: coarse)").matches;
+                const maxTouchPoints = ('ontouchstart' in window) || (navigator.maxTouchPoints > 0);
+                return matchMedia || maxTouchPoints;
+            } finally {
+                return false;
+            }
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)